src/tree/updater_quantile_hist.h

```diff
@@ -331,6 +333,9 @@ class QuantileHistMaker: public TreeUpdater {
     std::vector<NodeEntry> snode_;
     /*! \brief culmulative histogram of gradients. */
     HistCollection hist_;
+    /*! \brief culmulative local parent histogram of gradients. */
+    HistCollection phist_local_;
```

Comment:
prefix "p" means pointer. So, ```hist_local_```, probably, is better naming.
Or even ```hist_local_worker_```

Reply:
renamed, thanks